### PR TITLE
November 6 Filipino content buckets (Learn pages, Learn hub & guides)

### PR DESCRIPTION
**Pull request by corwintines**
## Description

- November 6 Filipino content buckets (Learn pages, Learn hub & guides)
- support images in markdown for 3 letter lang codes

### CODE REVIEW

File: src/content/translations/fil/guides/how-to-create-an-ethereum-account/index.md [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+
+Ang wallet ay isang app na tumutulong sa iyong pamahalaan ang iyong Ethereum account. Ginagamit nito ang iyong mga key para magpadala at tumanggap ng mga transaksyon at mag-sign in sa mga app. Maraming iba't ibang wallet na mapagpipilian—mobile, desktop, o kahit mga browser extension.
+
+<ButtonLink to="/wallets/find-wallet/">
+  Maghanap ng wallet
+</ButtonLink>
+
+Kung bago ka, maaari mong piliin ang filter na “Baguhan sa crypto” sa page na "maghanap ng wallet" upang matukoy ang mga wallet na naglalaman ng lahat ng kinakailangang feature na angkop para sa mga baguhan.
+
+![pagpili ng filter sa page na 'maghanap ng wallet'](./wallet-box.png)
```
Comment (wackerow) [inline src/content/translations/fil/guides/how-to-create-an-ethereum-account/index.md:23]: image

Images not loading

Comment (wackerow) [reply src/content/translations/fil/guides/how-to-create-an-ethereum-account/index.md:23]: Uncertain why... syntax is identical to other locales that are currently working just fine. Merge updated with `dev` and will let it rebuild first before moving further

Comment (wackerow) [reply src/content/translations/fil/guides/how-to-create-an-ethereum-account/index.md:23]: @corwintines @pettinarip Rebuilt and this is still showing up this way... any idea what could be going on here?

Comment (corwintines) [reply src/content/translations/fil/guides/how-to-create-an-ethereum-account/index.md:23]: I think this has to do with `gatsby-remark-fix-static-urls` only doing 2 letter lang codes. Let me look at this, think I know whats going on.

**Review by wackerow — APPROVED** [review]
Strange that this image issue is still showing on the preview deploy but not locally... the regex change should catch it, and local console logging shows the logic seems to be working. I'm going to pull this in to not let one image block the whole PR... will check `dev` preview deploy once in and if bug persists will open an issue.

**Pull request merged**